control-plane/api-gateway/common/helm_config.go

```diff
@@ -14,13 +14,15 @@ const componentAuthMethod = "k8s-component-auth-method"
 // This is a combination of the apiGateway stanza and other settings that impact api-gateways.
 type HelmConfig struct {
 	// ImageDataplane is the Consul Dataplane image to use in gateway deployments.
-	ImageDataplane             string
+	ImageDataplane string
+	// ImageConsulK8S is the Consul Kubernetes image to use in gateway deployments.
```

Comment:
This might be a little more clear since the docker image is `hashicorp/consul-k8s-control-plane`
```suggestion
	// ImageConsulK8S is the Consul Kubernetes Control Plane image to use in gateway deployments.
```